src/ProjectOrigin.WalletSystem.IntegrationTests/ApiTests.open_api_specification_not_changed.verified.txt

```diff
@@ -30,27 +33,32 @@
         ],
         "responses": {
           "200": {
-            "description": "Success",
+            "description": "Returns the aggregated claims.",
             "content": {
               "application/json": {
                 "schema": {
                   "$ref": "#/components/schemas/GranularCertificateResultList"
                 }
               }
             }
+          },
+          "401": {
+            "description": "If the user is not authenticated."
           }
         }
       }
     },
-    "/api/v1/claims": {
+    "/v1/claims": {
       "get": {
         "tags": [
           "Wallet"
         ],
+        "summary": "Gets all claims in the wallet",
         "parameters": [
           {
             "name": "start",
             "in": "query",
+            "description": "The start of the time range in Unix time.",
```

Comment:
We should write that it is in Unix time **seconds** - otherwise, someone will use Unix time **milliseconds**

Reply:
done